### PR TITLE
437 refactor cause card

### PR DESCRIPTION
**Pull request by azure-may**
### What does this PR do?
Refactored cause card to calculate days to go by adding date-fns package and percentage towards goal by calculating the amount raised towards the target amount. Added a  style tag to the progress bar div to adjust based upon percentage towards the target, set default width of div in SCSS file to 0. Adjusted enzyme tests as required.

#### How should this be manually tested?

1. On the terminal, run command ...
yarn test

### CODE REVIEW

**Review by aonomike — APPROVED** [review]
Looks good to me @azure-may. Thanks so much for working on this!

File: src/components/Causes/CauseCard/CauseCard.js [hunk 1]
```diff
@@ -20,12 +21,12 @@ const CauseCard = props => {
         </span>
         <div className="cause-card-progress">
           <div className="cause-card-progress-bar">
-            <div className="cause-card-progress-bar-active" />
+            <div className="cause-card-progress-bar-active" style={{ width: `${progress}%` }} />
           </div>
-          <p>17% of $30,000</p>
+          <p>{progress}% of ${props.targetAmount}</p>
```
Comment (aonomike) [inline src/components/Causes/CauseCard/CauseCard.js:26]: Niice, I like..

**Review by javpet — APPROVED** [review]
Great addition @azure-may !

**Comment by azure-may** [comment]
Thank you Mike and Peter!

**Pull request merged**